build/BuildFrameworksTasks/BuildNativeDependenciesTask.cs

```diff
@@ -1,60 +1,112 @@
 [TaskName("Build Native Dependencies")]
 public sealed class BuildNativeDependenciesTask : FrostingTask<BuildContext>
 {
     public override void Run(BuildContext context)
+    {
+        BuildSDL2(context);
+        BuildFAudio(context);
+    }
+
+    private void BuildSDL2(BuildContext context)
     {
         var sdlSourceDir = "native/monogame/external/sdl2/sdl";
         var sdlBuildDir = System.IO.Path.Combine(sdlSourceDir, "build");
 
-        if (context.DirectoryExists(sdlBuildDir))
-        {
-            context.DeleteDirectory(sdlBuildDir, new DeleteDirectorySettings { Recursive = true });
-        }
-        context.CreateDirectory(sdlBuildDir);
+        RecreateDirectory(context, sdlBuildDir);
+
+        var configureArgs = new ProcessArgumentBuilder()
+            .Append("-S").AppendQuoted(context.MakeAbsolute(new DirectoryPath(sdlSourceDir)).FullPath)
+            .Append("-B").AppendQuoted(context.MakeAbsolute(new DirectoryPath(sdlBuildDir)).FullPath)
+            .Append("-DSDL_STATIC=ON")
+            .Append("-DSDL_TEST=OFF");
+
+        AppendPlatformCMakeArgs(configureArgs, context, isSDL: true);
+
+        RunCMake(context, configureArgs, "SDL2 CMake configuration failed!");
+
+        RunCMakeBuild(context, sdlBuildDir, "Release", "SDL2 build failed!");
+    }
+
+    private void BuildFAudio(BuildContext context)
+    {
+        var faudioSourceDir = "native/monogame/external/faudio";
+        var faudioBuildDir = System.IO.Path.Combine(faudioSourceDir, "build");
+
+        RecreateDirectory(context, faudioBuildDir);
+
+        var sdlIncludeDir = System.IO.Path.Combine("native/monogame/external/sdl2/sdl", "include");
 
-        var configureSettings = new ProcessSettings { WorkingDirectory = sdlBuildDir };
-        var configureArgs = new ProcessArgumentBuilder();
-        // Add the relative path to the source directory.
-        configureArgs.Append("../");
-        configureArgs.Append("-DSDL_STATIC=ON -DSDL_TEST=OFF");
+        var configureArgs = new ProcessArgumentBuilder()
+            .Append("-S").AppendQuoted(context.MakeAbsolute(new DirectoryPath(faudioSourceDir)).FullPath)
+            .Append("-B").AppendQuoted(context.MakeAbsolute(new DirectoryPath(faudioBuildDir)).FullPath)
+            .Append("-DBUILD_SHARED_LIBS=OFF")
+            .Append($"-DCMAKE_C_STANDARD_INCLUDE_DIRECTORIES={context.MakeAbsolute(new DirectoryPath(sdlIncludeDir))}")
+            .Append($"-DCMAKE_CXX_STANDARD_INCLUDE_DIRECTORIES={context.MakeAbsolute(new DirectoryPath(sdlIncludeDir))}")
+            .Append("-DBUILD_SDL3=OFF");
 
-        // Append platform-specific CMake arguments.
+        AppendPlatformCMakeArgs(configureArgs, context, isSDL: false);
+
+        RunCMake(context, configureArgs, "FAudio CMake configuration failed!");
+
+        RunCMakeBuild(context, faudioBuildDir, "Release", "FAudio build failed!");
+    }
+
+    private void AppendPlatformCMakeArgs(ProcessArgumentBuilder args, BuildContext context, bool isSDL)
```

Comment:
For future proofing perhaps we can make this bool and Enum instead?
 for example something like
```csharp
public enum BuildTarget {
   SDL,
   FAudio
}
```

Otherwise if we add yet another native dependency (which we probably will) we will need to refactor this to either add yet another argument or change this anyway.